Sources/TemplateKit/Pipeline/TemplateSerializer.swift

```diff
@@ -224,6 +224,10 @@ public final class TemplateSerializer {
                     return .data(data)
                 }
             }
+            
+            guard !(data.isNull) else {
```

Comment:
we should omit the parens here.

Reply:
Done at b7b2b6dbd03dc0766f38962daddc0ca51e7e9c7c